packages/components/src/checkbox-control/style.scss

```diff
@@ -1,3 +1,76 @@
 .components-checkbox-control__input[type="checkbox"] {
-	margin-top: 0;
+	border: 1px solid #b4b9be;
+	background: #fff;
+	color: #555;
+	clear: none;
+	cursor: pointer;
+	display: inline-block;
+	line-height: 0;
+	height: 16px;
+	margin: 0 4px 0 0;
+	outline: 0;
+	padding: 0 !important;
+	text-align: center;
+	vertical-align: middle;
+	width: 16px;
+	min-width: 16px;
+	-webkit-appearance: none;
+	box-shadow: inset 0 1px 2px rgba(0, 0, 0, 0.1);
+	transition: 0.05s border-color ease-in-out;
+
+	&:focus {
+		border-color: #5b9dd9;
+		box-shadow: 0 0 2px rgba(30, 140, 190, 0.8);
+		/* Only visible in Windows High Contrast mode */
```

Comment:
The `/* */` comments are rendered in the compiled CSS, whereas `//` comments are stripped out. In this case it may actually be a good call to include these even in the compiled CSS because the transparent outline can look like a mistake otherwise.

Reply:
```suggestion
		// Only visible in Windows High Contrast mode.
```